ignite/handlers/checkpoint.py

```diff
@@ -101,32 +104,52 @@ class Checkpoint(Serializable):
 
     Note:
         This class stores a single file as a dictionary of provided objects to save.
-        The filename has the following structure: `{filename_prefix}_{name}_{suffix}.{ext}` where
+        The filename is defined by ``filename_pattern`` and by default has the following
+        structure: ``{filename_prefix}_{name}_{suffix}.{ext}`` where
 
         - ``filename_prefix`` is the argument passed to the constructor,
         - `name` is the key in ``to_save`` if a single object is to store, otherwise `name` is "checkpoint".
-        - `suffix` is composed as following `{global_step}_{score_name}={score}`.
+        - `suffix` is composed as following ``{global_step}_{score_name}={score}``.
+
+    +----------------+------------+-----------------------+----------------------------------------------+
+    | score_function | score_name | global_step_transform |  suffix                                      |
+    +================+============+=======================+==============================================+
+    |      None      |   None     |        None           | ``{engine.state.iteration}``                 |
+    +----------------+------------+-----------------------+----------------------------------------------+
+    |       X        |   None     |        None           | ``{score}``                                  |
+    +----------------+------------+-----------------------+----------------------------------------------+
+    |       X        |   None     |         X             | ``{global_step}_{score}``                    |
+    +----------------+------------+-----------------------+----------------------------------------------+
+    |       X        |    X       |         X             | ``{global_step}_{score_name}={score}``       |
+    +----------------+------------+-----------------------+----------------------------------------------+
+    |      None      |   None     |         X             | ``{global_step}``                            |
+    +----------------+------------+-----------------------+----------------------------------------------+
+    |       X        |    X       |        None           | ``{score_name}={score}``                     |
+    +----------------+------------+-----------------------+----------------------------------------------+
+
+    Above `global_step` defined by the output of `global_step_transform` and `score` defined by the output
+    of `score_function`.
+
+    By default, none of ``score_function``, ``score_name``, ``global_step_transform`` is defined, then suffix is
```

Comment:
Double quoted ``score_function`` should be like single quoted `score_function`, shouldn’t it ? Same for others.